public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -219,6 +219,11 @@ export class LokiDatasource
           return undefined;
         }
 
+        const dropErrorExpression = `${expr} | drop __error__`;
+        if (isQueryWithError(this.interpolateString(dropErrorExpression, placeHolderScopedVars)) === false) {
```

Comment:
Just a small safeguard that we're not breaking any query with that. Not really sure when and how that could happen though.

Reply:
Very nice 👌